builder/triton/access_config.go

```diff
@@ -180,7 +179,7 @@ func (c *Client) Compute() (*compute.ComputeClient, error) {
 func (c *Client) Network() (*network.NetworkClient, error) {
 	networkClient, err := network.NewClient(c.config)
 	if err != nil {
-		return nil, errwrap.Wrapf("Error Creating Triton Network Client: {{err}}", err)
```

Comment:
Hm, was that an okay error wrapping ? 🤔

Reply:
def LGTM: https://github.com/hashicorp/errwrap

Reply:
Maybe 🤔  the linter suggested replacing with fmt.Errorf because this one is deprecated

Reply:
ah no the linter is right !